ref/glossary.md

```diff
@@ -301,7 +301,9 @@ See [Hooks](https://discourse.ubuntu.com/t/hooks/28555).
 
 The base for an instance, which contains all necessary components like Anbox or the Android root file system. Anbox Cloud provides images based on different Android and Ubuntu versions and different architectures.
 
-See https://discourse.ubuntu.com/t/manage-images/17758 and https://discourse.ubuntu.com/t/provided-images/24185.
+The images can be an Anbox Cloud AOSP image which is based on the Android Open Source Project (AOSP), an operating system typically used in mobile devices or an Anbox Cloud AAOS image which is based on the Android Automotive OS (AAOS), an infotainment platform used in automobiles.
```

Comment:
One question: is it okay to mention AAOS image here? As far as I can see in our documentation, we haven't referenced AAOS elsewhere, indicating that we haven't officially announced support for AAOS in Anbox Cloud yet AFAIUC.

Reply:
I think it doesn't hurt but if we want to be technically correct we should skip this till 1.21 is out.

Reply:
I can mention enablement work for AAOS without actually mentioning the details. That just shows our roadmap.

Reply:
We mention a bug fix for `android.hardware.automotive.audiocontrol` service getting accidentally launched. So I wanted to set the stage for explaining AOSP vs AAOS images. Since we have it in our roadmap, I think it does not become a deal breaker if we include this in the glossary.

Reply:
As discussed in SU, working around this to not mention AOSP